bundler/lib/bundler/templates/newgem/README.md.tt

```diff
@@ -8,11 +8,13 @@ TODO: Delete this and the text above, and describe your gem
 
 Install the gem and add to the application's Gemfile by executing:
 
-    $ bundle add <%= config[:name] %>
+    $ bundle add UPDATE_WITH_YOUR_GEM_NAME_PRIOR_TO_RELEASE_TO_RUBYGEMS_ORG
 
 If bundler is not being used to manage dependencies, install the gem by executing:
 
-    $ gem install <%= config[:name] %>
+    $ gem install UPDATE_WITH_YOUR_GEM_NAME_PRIOR_TO_RELEASE_TO_RUBYGEMS_ORG
+
+TODO: Replace the `UPDATE_WITH_YOUR_GEM_NAME_PRIOR_TO_RELEASE_TO_RUBYGEMS_ORG` with your gem name before releasing it to RubyGems.org or delete this section if you do not plan to release to RubyGems.org. Please do not do it earlier due to security reasons.
```

Comment:
In next section TODO is at top of the section. Would it make sense to follow and do the same here (move TODO to top of this section)?

Reply:
@mensfeld ?

Reply:
I moved this todo up as well as the one from the first section to align them